content/en/api/integrations_gcp/gcp_create.md

```diff
@@ -1,31 +1,29 @@
 ---
-title: Google Cloud Platform
+title: Create a GCP integration
 type: apicontent
-order: 15.3
-external_redirect: /api/#google-cloud-platform
+order: 17.2
+external_redirect: /api/#create-a-gcp-integration
 ---
 
-## Google Cloud Platform
+## Create a GCP integration
 
-Configure your Datadog-Google Cloud Platform integration directly through the Datadog API.  
-[Read more about Datadog-Google Cloud Platform integration][1].
+Create a Datadog-Google Cloud Platform integration.
 
-**Note**: 
+**Note**:
 
-* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization. 
-
-* Using the `PUT` method updates the existing integration configuration in your Datadog organization.
+* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization.
```

Comment:
I'm a bit unclear here - does this add an additional config file, or does it combine your existing config with the new one?

Reply:
Yup this add a new configuration file, the `PUT` method would combine the config pushed with the existing configuration if any.